src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -660,27 +660,7 @@ public override string ToString()
 
         public override bool Equals(object obj)
         {
-            if (obj is Color)
-            {
-                Color right = (Color)obj;
-                if (value == right.value
-                    && state == right.state
-                    && knownColor == right.knownColor)
-                {
-                    if (name == right.name)
-                    {
-                        return true;
-                    }
-
-                    if (ReferenceEquals(name, null) || ReferenceEquals(right.name, null))
-                    {
-                        return false;
-                    }
-
-                    return name.Equals(name);
-                }
-            }
-            return false;
+            return obj is Color && this == (Color)obj;
```

Comment:
> Are we okay with that?

I think we need to be, and we'd ideally fix it in desktop, too.  It's fundamentally giving the wrong answer :)